src/components/match-layout/match-layout.tsx

```diff
@@ -1,31 +1,26 @@
 import { Container } from "@mui/material";
-import { type ReactNode, type FC } from "react";
-
-import { SplashScreenChat } from "~/components/splash-screen-chat/index.js";
-import { SplashScreenVoting } from "~/components/splash-screen-voting/index.js";
-
+import { type FC, type ReactNode } from "react";
+import { type PlayerType } from "~/types/index.js";
+import { Interstitials } from "~/components/interstitials/index.js";
 import { styles } from "./styles.js";
 
 interface Props {
   splashScreenVariant?: "chat" | "voting";
```

Comment:
Tip: if you want this to be based on `MatchStage` you can use `Pick` https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys